tests/backup/backup_test.go

```diff
@@ -786,10 +786,9 @@ var _ = Describe("{ShareBackupWithUsersAndGroups}", func() {
 		}
 	})
 	It("Share large number of backups", func() {
-		Step("Validate applications and get their labels", func() {
+		Step("Validate applications", func() {
 			log.InfoD("Validate applications and get their labels")
```

Comment:
It is only validating application not getting labels

Reply:
Done